src/App/components/Header/components/AddLocation/components/DrawerBlock/DrawerBlock.tsx

```diff
@@ -126,6 +126,8 @@ const DrawerBlock: React.FC<IDrawerBlockProps> = ({
                 </div>
               </MenuItem>
             ))
+          ) : searchText === '' ? (
+            <div></div>
           ) : (
             <div className={style.notFound}>{t('AddLocation.NotFound')}</div>
           )}
```

Comment:
It is better to use separate component or split the logic to the function instead of using complex ternary operator it will increase the readability of the code.

Or, you just can use like: 
```
<>
...
{ !!locationsFound.length && locationsFound.map( .... ) }
{ !!locationsFound.length && searchText !== '' && div... }
...
</>
```